binaries/scheduler/config/local.local

```diff
@@ -10,7 +10,9 @@
   },
   "SchedulerConfig": {
     "Type": "stateful",
-    "MaxRetriesPerTask" : 2
+    "MaxRetriesPerTask" : 2,
+    "DebugMode" : false,
+    "RecoverJobsOnStartup" : false
```

Comment:
can you put a comma at the end of this line? - then when we add a new configuration, we won't have to add a comma to the existing one.